code/modules/clothing/masks/gasmask.dm

```diff
@@ -74,6 +74,7 @@
 	options["The Madman"] = "joker"
```

Comment:
There's already a 'madman'

Reply:
oh right I'd didn't notice that I'll fix it in a bit